src/writer_postgres.py

```
@@ -3,32 +3,36 @@
 
 import boto3
 from botocore.exceptions import ClientError
+
 try:
     import psycopg2  # noqa: F401
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)